slack_lambda/main.tf

```diff
@@ -7,27 +7,84 @@ locals {
     import urllib3
     import json
     import os
+    import re
+    import datetime
+
     ssm = boto3.client('ssm')
     slackChannel = os.environ['slack_channel']
     slackUsername = os.environ['slack_username']
     slackIcon = os.environ['slack_icon']
     slackUrlParam = os.environ['slack_webhook_url_parameter']
     parameter = ssm.get_parameter(Name=slackUrlParam, WithDecryption=True)
     http = urllib3.PoolManager()
+
     def lambda_handler(event, context):
         url = parameter['Parameter']['Value']
         eventmsg = event['Records'][0]['Sns']['Message']
+        blocks = null
         try:
           data = json.loads(eventmsg)
           if 'detailType' in data and data['detailType'] == 'CodePipeline Pipeline Execution State Change':
             msgtext = 'auto-terraform:  ' + data['detail']['pipeline'] + ' pipeline ' + data['detail']['state'] + ' with execution ID ' + data['detail']['execution-id']
           elif 'AlarmName' in data and 'AlarmDescription' in data:
+            blocks = [
+              "type": "section",
```

Comment:
I tried running this by directly editing https://us-west-2.console.aws.amazon.com/lambda/home?region=us-west-2#/functions/snstoslack_login_otherevents?tab=code, and this line failed with a syntax error. I don't know Python super well, so I'm not sure if it's an issue with the dictionary syntax or the list.

Reply:
that a syntax error... missing some `{` 🤡

Reply:
fixed in c38b612b8eec480087272f3071a833d99e5dee9e, going to give that page a shot